src/ucp/core/ucp_mm.c

```diff
@@ -296,6 +368,19 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh)
     return status;
 }
 
+ucs_status_t ucp_mem_query(const ucp_mem_h memh, ucp_mem_attr_t *attr)
+{
+    if (memh->address) {
+        attr->field_mask = UCP_MEM_ATTR_FIELD_ADDRESS |
```

Comment:
Ok, but I like UNSUPPORTED more from perspective that field_flags introduced for backward compatibility. This might be actual for ucp*params functions as well